src/subscriptionModules/eventScannerBased.ts

```diff
@@ -109,6 +109,8 @@ export class EventScannerBased implements ISubscriptionModule{
           } while (this.isNewScanRequired);
         } catch (error) {
           this.logger.error(`last SCAN had an issue !: ${JSON.stringify(error)}`)
+          // better to exit and restart
+          process.exit(-1)
```

Comment:
I believe this alone will be enough to trigger the restart alerts. Not sure what's the rationale for the restarter cronjob...

Reply:
this has been removed, was testing code

Reply:
the memory leak haven't been spotted yet, this restarter will assure a fresh start (i.e. every 6 hours)